### PR TITLE
sync manifests with Antrea v1.2.1

### PR DESCRIPTION
**Pull request by jwsui**
Sync manifests with Antrea v1.2.1 following below steps:
1. Copy antrea.yml into antrea-manifest/antrea.yml.
-- Replace the antrea-agent.conf/antrea-cni.conflist/antrea-controller.conf with template;
-- Replace host-cni-conf and host-cni-bin with template;
-- Replace Antrea image with template;
-- Add annotation 'release.openshift.io/version' for deployment antrea-controller and daemonset
   antrea-agent.
2. Update the antreainstall_cr under directory of deploy/kubernetes and deploy/openshift according
to the change in antrea.yml from Antrea repo.
3. Update the role.yaml under directory of deploy/kubernetess and deploy/openshift according to the
rbac change in antrea.yml

Also, update Dockerfile for specifying OVS and CNI version while building Antrea UBI image.

### CODE REVIEW

File: deploy/kubernetes/operator.antrea.vmware.com_v1_antreainstall_cr.yaml [hunk 1]
```diff
@@ -79,12 +82,23 @@ spec:
...
+    #defaultMTU: 0
 
     # Whether or not to enable IPsec encryption of tunnel traffic. IPsec encryption is only supported
     # for the GRE tunnel type.
     #enableIPSecTunnel: false
 
+    # ClusterIP CIDR range for Services. It's required when AntreaProxy is not enabled, and should be
+    # set to the same value as the one specified by --service-cluster-ip-range for kube-apiserver. When
+    # AntreaProxy is enabled, this parameter is not needed and will be ignored if provided.
+    #serviceCIDR: 10.96.0.0/12
```
Comment (dantingl) [inline deploy/kubernetes/operator.antrea.vmware.com_v1_antreainstall_cr.yaml:94]: I don't think we need `serviceCIDR` in install CR, it is configured in cluster network CR

File: deploy/kubernetes/operator.antrea.vmware.com_v1_antreainstall_cr.yaml [hunk 2]
```diff
@@ -79,12 +82,23 @@ spec:
     # Default MTU to use for the host gateway interface and the network interface of each Pod.
     # If omitted, antrea-agent will discover the MTU of the Node's primary interface and
     # also adjust MTU to accommodate for tunnel encapsulation overhead (if applicable).
-    #defaultMTU: 1450
+    #defaultMTU: 0
```
Comment (dantingl) [inline deploy/kubernetes/operator.antrea.vmware.com_v1_antreainstall_cr.yaml:85]: Can you confirm in Antrea what's the default value if this is set 0 in ConfigMap, we need to update network CR status for MTU

File: deploy/kubernetes/operator.antrea.vmware.com_v1_antreainstall_cr.yaml (hunk 1 above)
Comment (jwsui) [reply deploy/kubernetes/operator.antrea.vmware.com_v1_antreainstall_cr.yaml:94]: > I don't think we need `serviceCIDR` in install CR, it is configured in cluster network CR

yes, for openshift we don't need it. But for kubernetes, we'll keep it as there is no cluster network CR, right?

File: deploy/kubernetes/operator.antrea.vmware.com_v1_antreainstall_cr.yaml (hunk 2 above)
Comment (jwsui) [reply deploy/kubernetes/operator.antrea.vmware.com_v1_antreainstall_cr.yaml:85]: > Can you confirm in Antrea what's the default value if this is set 0 in ConfigMap, we need to update network CR status for MTU

There will be a new patch to find a way to get MTU and update MTU in network status CR.

**Review by dantingl — APPROVED** [review]

**Pull request merged**